boards/esp32-wroom-32/Makefile.dep

```diff
@@ -7,7 +7,7 @@ ifneq (, $(filter enc28j60, $(USEMODULE)))
 
     # avoid multiple definitions when package depenedencies are resolved recursively
     ifndef ENC28J60_PARAM_DEFINED
-        export ENC28J60_PARAM_DEFINED = 1
```

Comment:
Just to be sure @gschorcht there is not weird use case for why the export was needed?

Reply:
Seems to work without the export. Thanks for cleaning up.